### PR TITLE
Resolve missing types "Boolean", "String"

### LINKED ISSUE #1295: Could not fetch manifest (Access denied.)
Hey, there! Thanks for all the hard work you all put in. I'm having trouble getting my Operation Registry up and running.

**Intended outcome:**
Start API & `apollo-server-plugin-operation-registry` starts blocking unregistered operations.

**Actual outcome:**
```
Apollo Server could not begin serving requests immediately because the operation manifest could not be fetched.  Attempts will continue to fetch the manifest, but all requests will be forbidden until the manifest is fetched. Could not fetch manifest <?xml version='1.0' encoding='UTF-8'?><Error><Code>AccessDenied</Code><Message>Access denied.</Message><Details>Anonymous caller does not have storage.objects.get access to engine-op-manifest-storage-prod/71a8658c58f3edfbf807fb48352823488d8da9801d85c295714950c290e07935d8a43b33f31614aaf2d1a6f7f045b83f8a58c8d3bb6f31d45fd10da5d9186e37/9327c42b61236bf40adfaa5ca4f60a5a101ab91813235d2c7f1a0eff26abaf14d4244bafd892c2d32fd1ae8aa5088c4e6c555bc88f8a32892f12f410ef75d41a.v2.json.</Details></Error>
```

I fear this may have to do with my account. Not sure about that, though. Could also be related to #996 

I've already tried re-pushing both client and server with `apollo client:push` and `apollo service:push`. Both succeed:

**Client:**
```
Successfully pushed 38 operations from the WebApp client to the thegospelfund service in Engine
```

**Server:**
```
id      schema         tag
──────  ─────────────  ───────
35f5cb  thegospelfund  current
```

**Versions are as follows:**
apollo-server-plugin-operation-registry: `0.1.0-alpha.4`
apollo CLI: `apollo/2.12.3`

### LINKED ISSUE #1296: 2.12.3: Error: Cannot use GraphQLSchema "[object GraphQLSchema]" from another module or realm
After upgrading from `apollo@2.12.2` --> `apollo@2.12.3`, I started receiving the following error while trying to download my schema:

```
> apollo service:download --endpoint=$API_ENDPOINT_PROXY/v2/graphql graphql-schema.json

  ✔ Loading Apollo Project
  ✖ Saving schema to graphql-schema.json
    → spurious results.
Error: Cannot use GraphQLSchema "[object GraphQLSchema]" from another module or realm.
Ensure that there is only one instance of "graphql" in the node_modules
directory. If different versions of "graphql" are the dependencies of other
relied on modules, use "resolutions" to ensure only one version is installed.
https://yarnpkg.com/en/docs/selective-version-resolutions
Duplicate "graphql" modules cannot be used at the same time since different
versions may have different capabilities and behavior. The data from one
version used in the function from another could produce confusing and
spurious results.
    at instanceOf (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/node_modules/graphql/jsutils/instanceOf.js:37:13)
    at isSchema (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/node_modules/graphql/type/schema.js:34:34)
    at assertSchema (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/node_modules/graphql/type/schema.js:38:4)
    at validateSchema (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/node_modules/graphql/type/validate.js:53:28)
    at assertValidSchema (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/node_modules/graphql/type/validate.js:77:16)
    at assertValidExecutionArguments (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/node_modules/graphql/execution/execute.js:146:35)
    at executeImpl (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/node_modules/graphql/execution/execute.js:67:3)
    at execute (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/node_modules/graphql/execution/execute.js:62:256)
    at Object.introspectionFromSchema (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/node_modules/graphql/utilities/introspectionFromSchema.js:38:37)
    at Task.task (~/Sites/nextradata/mmdp/src/mmdp-app/node_modules/apollo/lib/commands/service/download.js:21:82)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! mmdp-app@0.0.7 update:schema: `apollo service:download --endpoint=$API_ENDPOINT_PROXY/v2/graphql graphql-schema.json`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the mmdp-app@0.0.7 update:schema script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/joncursi/.npm/_logs/2019-05-23T20_52_56_722Z-debug.log
ERROR: "update:schema" exited with 2.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! mmdp-app@0.0.7 update: `dotenv -e .env -- npm-run-all update:schema update:types`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mmdp-app@0.0.7 update script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/joncursi/.npm/_logs/2019-05-23T20_52_56_754Z-debug.log
```

Downgrading back to `2.12.2` causes the error to go away.

### PR DESCRIPTION
**Pull request by trevor-scheer**
## Preface
A change to graphql's `buildClientSchema` removed some behavior that we depended on. Previously, with [caching in place](https://github.com/graphql/graphql-js/pull/1677/files#diff-768f1266aca135746881bd4d9ea9093eL97), basic types (Boolean, String, ...) were returned even if they didn't exist in typeMap. The "fix" was itself a breaking change, adding basic types that weren't previously added to the typeMap.

Full discussion here: https://github.com/graphql/graphql-js/commit/183ff32bee4bc23eb23657f79f414c2400ecb06a#r32971387

## Chain of events
* This commit lands in the `14.2.0` release, breaking our usage of `buildClientSchema`
https://github.com/graphql/graphql-js/pull/1677
* A follow-up fix is released in `14.2.1`, which is itself a breaking change
https://github.com/graphql/graphql-js/pull/1808
* The breaking change is noted in the discussion above^, resulting in the release of `14.3.1` with this fix:
https://github.com/graphql/graphql-js/pull/1809

## Approach / Discussion
@justinanastos and I discovered with some digging that we depended on `buildClientSchema` to fill in the missing built-in types for us (we were handing off an incomplete schema, but it would correct our error). We realized that we can request the built-in types and solve the problem on our end by passing a complete schema.

This PR addresses a couple issues that stem from the ongoing graphql issue (and subsequent fixes):
1) We can request the built in types from Engine so they won't be missing to begin with
2) Disallow versions of graphql that include the broken fix `graphql@14.2.1 - 14.3.0`.

### Why exclude the range `14.2.1 - 14.3.0`?
The broken fix causes a regression for service:push. Through the version range of the broken fix, ALL built in types are included during `buildClientSchema` (even if they're not part of the schema). This causes a hash mismatch between the pushed schema and the schema hash that Apollo Server recognizes.

Fixes #1296
Fixes #1295

TODO:

- [ ] Update CHANGELOG.md\* with your change (include reference to issue & this PR)
- [ ] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests and linter rules pass

\*Make sure changelog entries note which project(s) has been affected. See older entries for examples on what this looks like.

### CODE REVIEW

File: packages/apollo/package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "git-parse": "1.0.3",
     "git-rev-sync": "1.12.0",
     "glob": "7.1.4",
-    "graphql": "~14.2.1",
+    "graphql": "14.0.2-14.2.0 || ^14.3.1",
```
Comment (abernix) [inline packages/apollo/package.json]:
```suggestion
    "graphql": "14.0.2 - 14.2.0 || ^14.3.1",
```

File: packages/apollo-language-server/package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "cosmiconfig": "^5.0.6",
     "dotenv": "^8.0.0",
     "glob": "^7.1.3",
-    "graphql": "~14.2.1",
+    "graphql": "14.0.2-14.2.0 || ^14.3.1",
```
Comment (abernix) [inline packages/apollo-language-server/package.json]:
```suggestion
    "graphql": "14.0.2 - 14.2.0 || ^14.3.1",
```

File: packages/apollo/package.json (hunk 1 above)
Comment (trevor-scheer) [reply packages/apollo/package.json]: Appreciate the inline suggestion, but I went ahead and resolved in a commit before I could see them ;P

Resolved in 363a01f

File: packages/apollo-language-server/package.json (hunk 2 above)
Comment (trevor-scheer) [reply packages/apollo-language-server/package.json]: Same as above, resolved in 363a01f

**Pull request merged**